Small patch to remove mc13783 stuff from alsa driver building for i.MX27

Index: linux-2.6.29/sound/arm/Makefile
===================================================================
--- linux-2.6.29.orig/sound/arm/Makefile	2009-03-31 10:03:59.000000000 +0200
+++ linux-2.6.29/sound/arm/Makefile	2009-03-31 10:07:16.000000000 +0200
@@ -18,8 +18,11 @@
 obj-$(CONFIG_SND_PXA2XX_AC97)	+= snd-pxa2xx-ac97.o
 snd-pxa2xx-ac97-objs		:= pxa2xx-ac97.o
 
-snd-mc13783-objs		:= imx2-pcm.o imx2-ssi.o imx2-dam.o mc13783.o devdma.o
-obj-$(CONFIG_SND_IMX2_PCM)	+= snd-mc13783.o
+#snd-mc13783-objs		:= imx2-pcm.o imx2-ssi.o imx2-dam.o mc13783.o devdma.o
+#obj-$(CONFIG_SND_IMX2_PCM)	+= snd-mc13783.o
+
+obj-$(CONFIG_SND_IMX2_PCM)     += snd-mx2.o
+snd-mx2-objs               := imx2-ssi.o imx2-dam.o devdma.o
 
 obj-$(CONFIG_SND_IMX_TSC2102) += snd-imx-alsa-tsc2102.o
 snd-imx-alsa-tsc2102-objs :=   imx-alsa.o imx-alsa-tsc2102.o imx-alsa-tsc2102-mixer.o
